examples/html-ethers-5/src/main.js

```diff
@@ -0,0 +1,36 @@
+import { createWeb3Modal, defaultConfig } from '@web3modal/ethers-5'
+
+// @ts-expect-error 1. Get projectId
+const projectId = import.meta.env.VITE_PROJECT_ID
+if (!projectId) {
+  throw new Error('VITE_PROJECT_ID is not set')
+}
+
+// 2. Create wagmiConfig
+const chains = [1, 42161]
+
+// 3. Create modal
+const modal = createWeb3Modal({
+  ethersConfig: defaultConfig({
+    metadata: {
+      name: 'Web3Modal',
+      description: 'Web3Modal Laboratory',
+      url: 'https://web3modal.com',
+      icons: ['https://avatars.githubusercontent.com/u/37784886']
+    },
+    coinbaseDefaultChainId: 1,
```

Comment:
Why do we have coinbase explicit options? These should be `defaultChain` and `rpcUrl` and propagate to all default providers. Ideally, we stay as close to current options api as we can.